src/helm/common/test_logging.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import tempfile
+import shutil
+import textwrap
+import pathlib
+from helm.benchmark import run
+
+
+def test_run_with_custom_logging_config():
+    # Setup temporary directory
+    tmp_dir = pathlib.Path(tempfile.mkdtemp())
```

Comment:
nit: consider using the `tempfile.NamedTemporaryFile` context manager ([doc](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile)) which automatically deletes the file for you.

Reply:
I updated the tests to use context managers for both tempdir and sys.argv cleanup. (and also fixed a 3.9 mypy issue).

Do you want me to add the `--log-config` option to the other CLIs?

Reply:
Added them to everything except proxy/cli.py, which had a bunch of modal subparsers, and I'm not sure how used it is in HELM workflows. I figure we can add it later if needed.

Lets see if CI passes now

Reply:
Tests are fixed. Not really sure what proxy/cli.py is, removing it would be better in a different PR to keep this self-contained.